src/components/LandingPage.js

```diff
@@ -0,0 +1,180 @@
+    };
+  }
+
+  openOrCloseUserDropdown = (bol) => {
+    if (bol) {
+      this.setState({
+        showUserDropdown: true
+      });
+    } else {
+      this.setState({
+        showUserDropdown: false
+      });
+    };
+  }
+
+  render () {
+    const {
+      landingPage,
+      bootstrap
+    } = this.props;
+
+    const {
+      first_name,
+      authenticated,
+      login,
+      logout
+    } = bootstrap;
+
+    const {
+      fetchClientConfigs,
+      fetchDashboardJSONs,
+      dashboardJSONs
+    } = landingPage;
+    
+    return (
+      <div className={styles.LandingPage}>
+        <div className={styles.LandingPageBackground}>
+          <img src={backgroundImage} alt="background" className={styles.LandingPageBackgroundImage} />
+        </div>
+        <header>
+          <div className={styles.UserDropdown}>
+            {authenticated === true ? (
+              !this.state.showUserDropdown ? (
+                <div
+                  className={styles.UserDropdownClosed}
+                  onClick={() => this.openOrCloseUserDropdown(true)}
+                >
+                  <i className="fa fa-caret-down" />
+                  &nbsp;&nbsp;
+                  <i className="fa fa-user" />
+                  &nbsp;&nbsp;
+                  {first_name}
+                </div>
+              ) : (
+                <div
+                  className={styles.UserDropdownOpen}
+                  onClick={() => this.openOrCloseUserDropdown(false)}
+                >
+                  <i className="fa fa-caret-up" />
+                  &nbsp;&nbsp;
+                  <i className="fa fa-user" />
+                  &nbsp;&nbsp;
+                  {first_name}
+                  <div
+                    className={styles.UserDropdownContent}
+                  >
+                    <a href="https://sso.lizard.net/edit_profile/"
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      <i className="fa fa-pencil" />
+                      &nbsp;&nbsp;Edit&nbsp;Profile
+                    </a>
+                    <a href={logout}>
+                      <i className="fa fa-power-off" />
+                      &nbsp;&nbsp;Logout
+                    </a>
+                  </div>
+                </div>
+              )
+            ) : (
+              this.state.fetchSlugs === "RECEIVED" && this.state.fetchJSONS === "RECEIVED" ? (
+                <a href={login}>Login</a>
+              ) : null
+            )}
+          </div>
+          <h1 className={styles.LandingPageTitle}>Dashboards</h1>
+        </header>
+        <div className={styles.LandingPageBody}>
+          <div className={styles.DashboardList}>
+            {dashboardJSONs.map((dashboard, i)=>{
+              return (
+                <a 
+                  className={styles.Dashboard}
+                  key={i}
+                  href={`/${dashboard.slug.client_slug}`}
```

Comment:
Currently react router does not work between the landing page and the tile dashboards. This is one of the places where we should make a change if we want it to work.

Reply:
Yes, it's annoying that it reloads each time we navigate to a specific dashboard or go back to the landing page. Nice that you have already created a ticket for it. Thanks!